apps/server/src/modules/authentication/strategy/jwt-validation.adapter.ts

```diff
@@ -1,11 +1,18 @@
-import { Injectable } from '@nestjs/common';
-
+import { CACHE_MANAGER, Inject, Injectable } from '@nestjs/common';
+import { CacheService } from '@shared/infra/cache';
+import { CacheStoreType } from '@shared/infra/cache/interface/cache-store-type.enum';
+import { Cache } from 'cache-manager';
 import jwtWhitelist = require('../../../../../../src/services/authentication/logic/whitelist');
```

Comment:
I moved the imports to the imports-from-feathers.ts, but when building the docker image in github it throws an error, because it can't import from the test folder since it is not included in the image.
```
apps/server/src/imports-from-feathers.ts:3:38 - error TS2307: Cannot find module '../../../test/utils/redis/redisMock.js' or its corresponding type declarations.
```
Should I add the folder to the Dockerfile so that the import can be resolved?

Reply:
Ah I see. Please use the direct path in the spec file and remove the import from the test folder from imports-from-feathers.ts